### ISSUE TITLE
Bundled Test::More fails with perl 5.41.x

### ISSUE THREAD

**Issue by eserte** [pmqs/BerkeleyDB#12]
With perl 5.41.x (e.g. 5.41.4) the bundled Test::More does not work anymore:
```
"/tmp/basesmoker-reloperl-k83W/bin/perl" -MExtUtils::Command::MM -e 'cp_nonempty' -- BerkeleyDB.bs blib/arch/auto/BerkeleyDB/BerkeleyDB.bs 644
PERL_DL_NONLAZY=1 "/tmp/basesmoker-reloperl-k83W/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
String found where operator expected (Missing semicolon on previous line?) at t/Test/More.pm line 375, near "like($this, qr/that/, '"
  (Might be a runaway multi-line '' string starting on line 364)
syntax error at t/Test/More.pm line 375, near "like($this, qr/that/, '"
Execution of t/Test/More.pm aborted due to compilation errors.
Compilation failed in require at t/000prereq.t line 12.
BEGIN failed--compilation aborted at t/000prereq.t line 12.
# Looks like your test died before it could output anything.
t/000prereq.t .. 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
...
```

Probably best is to remove t/Test/* and instead put proper prereq specifications to Makefile.PL & META.*.

**Comment by pmqs** [comment]
Good point. Unbundled Test/More& Builder from the module.

**Comment by pmqs** [comment]
Fixed in 0.66.
